### ISSUE TITLE
Distutils needs to support distinct package namespaces

### ISSUE THREAD

**Issue by kayhayen** [Nuitka/Nuitka#571]
In `bdist_nuitka` right now, we pick a top level package, and include all packages below it, that fails the case of #527 and also doesn't work well when 2 packages are provided by one wheel.

For a wheel with packages names to be included:

```
[
    "something.a"
    "something.b"
    "something",
    "helper",
]
```

The result should not be one command, including them all, but instead, with multiple top level packages picked (here `something` and `helper`, multiple commands should be issued

```
# 2 commands 
--module something --include-package=something --include-package=something.a --include-package=something.b
--module helper 

-> something.pyd
-> helper.pyd
```

**Comment by kayhayen** [comment]
This should solve many issues that @jayvdb has raised for good. Tommy has worked on this in GSoC 2019 and knows what to do.

**Comment by kayhayen** [comment]
Once we have something workable, I guess that @jayvdb can put it through his OBS of all openSUSE again.

**Comment by kayhayen** [comment]
This was implemented for 0.6.6 by @tommyli3318
